### PR TITLE
RA-1756: Added logic to allow adding non-coded patient Condition

### PR DESCRIPTION
**Pull request by jnsereko**
**Issue:**  https://issues.openmrs.org/browse/RA-1756

I have improved code to allow saving of non-coded concepts.
cc  @isears @mozzy11 @dkayiwa @sherrif10

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js [hunk 1]
```diff
@@ -43,19 +43,13 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
         }
 
     self.validateCondition = self.validateCondition || function () {
-        var concept ;
-        if($scope.concept.concept.uuid) {
-            concept = $scope.concept.concept;
-        } else {
-            concept = $scope.condition.concept;
+        if ($scope.condition.concept===null||$scope.concept.concept==null){
```
Comment (HerbertYiga) [inline omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js:46]: Do we need these two comparisons here ? @jnsereko   have you tested these changes with a running instance of openmrs to see that all performs as required!

Comment (jnsereko) [reply omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js:46]: think the above if statement checks if its a concept(by having a uuid) and assigns a **Concept** to the `concept `variable or **Condition** to the `concept `variable.  

I have tested this using our demo by editing the **addcondition.controller.js** file using chrome tools

Comment (jnsereko) [reply omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js:46]: results 

https://user-images.githubusercontent.com/58003327/115373830-ea3df600-a1d4-11eb-85cd-1b30f4c0c786.mp4

**Comment by mseaton** [comment]
Closing this PR in favor of #468 .  See comments on RA-1756

**Pull request closed without merging**